doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -1438,3 +1441,5 @@ for completeness and assistance in understanding the code.
    ``arrayobject.h`` header. This type is not exposed to Python and
    could be replaced with a C-structure. As a Python type it takes
    advantage of reference- counted memory management.
+
+   .. c:type:: PyArrayMapIterObject
```

Comment:
This [looks a bit strange](https://16130-908607-gh.circle-artifacts.com/0/doc/build/html/reference/c-api/types-and-structures.html?highlight=types%20structures#c.PyArrayMapIterObject) without any descriptive text, but maybe that can be redone in a subsequent PR. But I think it should be dedented to mach the other types?

Reply:
For now `PyArrayMapIterObject` is not referenced anywhere else, so I think it is also possible to simply remove the reference above.

Reply:
sounds like a good idea.